ts/features/uaDonations/screens/UAWebViewScreen.tsx

```diff
@@ -81,30 +77,60 @@ const handleOnMessage = (
     JSON.parse(event.nativeEvent.data)
   );
   if (maybeMessage.isLeft()) {
-    handleError("decoding error: " + readableReport(maybeMessage.value));
+    void mixpanelTrack("UADONATIONS_WEBVIEW_DECODE_ERROR", {
+      reason: `decoding error: ${readableReport(maybeMessage.value)}`
+    });
+    handleError();
     return;
   }
   switch (maybeMessage.value.kind) {
     case "webUrl":
       const webUrl = maybeMessage.value.payload;
+      void mixpanelTrack("UADONATIONS_WEBVIEW_OPEN_WEBURL_REQUEST", {
+        url: webUrl
+      });
       openWebUrl(webUrl, () => {
-        handleError("webUrl error: " + webUrl);
+        void mixpanelTrack("UADONATIONS_WEBVIEW_OPEN_WEBURL_ERROR", {
+          url: maybeMessage.value.payload
+        });
+        handleError();
       });
       break;
     case "payment":
       const { nav, cf, amount } = maybeMessage.value.payload;
+      void mixpanelTrack("UADONATIONS_WEBVIEW_PAYMENT_REQUEST", {
+        organizationFiscalCode: cf,
+        paymentNoticeNumber: nav,
```

Comment:
```suggestion
        paymentNoticeNumber: PaymentNoticeNumberFromString.encode(nav),
```
The same reason of below